src/accordion.test.mouse.ts

```diff
@@ -70,7 +70,7 @@ test(
 );
 
 test(
-  'opens when clicked programmatically',
+  'opens when clicked via `click()`',
```

Comment:
Accordion and other components can be opened programmatically in two ways: via `click()` and by setting their `open` attributes. So I changed the name of tests like this to distinguish between the two.